src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/PackageManagerControl.xaml

```diff
@@ -108,6 +110,22 @@
               UpdateButtonClicked="PackageList_UpdateButtonClicked"
               Width="{Binding ActualWidth, ElementName=_leftSide}"/>
 
+            <vs:LiveTextBlock 
+              x:Name="LtbLoading" 
+              Visibility="Collapsed" 
+              IsFrequencyLimited="True"
+              Frequency="2">
+              <vs:LiveTextBlock.Text>
+                <MultiBinding 
+                    Converter="{StaticResource LiveTextLoadingConverter}"
+                    TargetNullValue="Nuuuuuuuul"
+                    FallbackValue="Faaaaaaaaail">
+                  <Binding RelativeSource="{RelativeSource FindAncestor, AncestorLevel=1, AncestorType={x:Type nuget:PackageManagerControl}}" Path="IsTabChanged" />
```

Comment:
@rrelyea @donnie-msft Am I doing right the bindings?

Reply:
it likely works, but using a datatrigger like here: https://stackoverflow.com/a/30172553/4317113 would likely be more natural...and you could avoid some custom code.

Reply:
Watch the Debug Output Window, if it can't find an Ancestor or something, it will tell you there and indicate which RelativeSource string failed.

Reply:
Changed to a code behind approach. Removed any Binding/triggers modifications.